docs/team/raydenlim.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default.md
+title: "Jian Song (rayden)'s Project Portfolio Page"
+---
+
+### Name: Rayden
+**Bio:** <br>
+* I am interested in cybersecurity
+* I believe that laughter is the best medicine
+
+### **Joke of the day**<br>
+**QNS :** Why did the programmer constantly run into bugs? <br>
+**ANS :** Because he used light mode, and bugs are attracted to the light.
+
+### Project: FAKEJARVIS
+![Logo](images/fakejarvis.png)
+
+FAKEJARVIS is a greenfield project built on Address Book 3 (AB3), with the intention to help CS1101S Avengers manage their Teaching Assistant (TA) duties.
+
+Given below are my contributions to the project.
+
+* **New Feature 1**:
+    * Highlights:
+
+    * Credits:
+
+* **New Feature 2**:
+
+* **Code contributed**: [RepoSense link]()
```

Comment:
I like that you tried to include the reposense link but I think it would be better to include the actual url to the reposense which can be found on the cs2103t website

Reply:
Ok!